assets/stylesheets/components/blog-teaser.scss

```diff
@@ -0,0 +1,19 @@
+.blog .content {
+  display: flex;
+  align-content: flex-start;
+  flex-wrap: wrap;
+
+  .blog-teaser {
+    background-color: red;
+
+    &:first-child {
+      background-color: pink;
+
+      // @media screen and (min-width: 930px) {
+      //   flex: 2 32em;
+      // }
+    }
+  }
+
+
+}
```

Comment:
Blog teaser styling in footer commit? Please try to keep PRs to one thing. Just open a new branch. If you need work in this branch for the other, base you new branch of the existing and just list the dependency in the comment.